documentation/modules/common-attributes.adoc

```diff
@@ -11,6 +11,7 @@
 :operator: mtv-operator
 :operator-name-ui: Migration Tookit for Virtualization Operator
 :operator-name: MTV Operator
+:osh: OpenStack
```

Comment:
for your consideration - we usually use the acronom `osp` (that stands for openstack platform), it would be better than `osh`

Reply:
good idea.